test/matrix/matrix.cpp

```diff
@@ -557,7 +557,7 @@ class Matrix : public CommonTestFixture {
     using Mtx = typename T::matrix_type;
     using index_type = typename Mtx::index_type;
     using value_type = typename Mtx::value_type;
-    using mixed_value_type = gko::next_precision<value_type>;
+    using mixed_value_type = gko::next_precision_base<value_type>;
```

Comment:
What about `gko::next_precision` here?

Reply:
I guess it doesn't matter, since `value_type` is always double. So I would prefer the non-base version.

Reply:
It matters when enabling dpcpp_single_mode, `next_precision_base<float>` will be double, which is what we test now, but `next_precision<float>` will be half